index.js

```diff
@@ -0,0 +1,43 @@
+import data from './data.js';
+
+const myFeatures = data.features;
+
+const menuBtn = document.querySelector('.hamburger');
+const menuLinks = document.querySelector('.nav-links');
+const closeMenuBtn = document.querySelector('.close-menu-btn button');
+const header = document.querySelector('header');
+
+window.addEventListener('scroll', () => {
+  if (window.pageYOffset > 100) {
+    header.classList.add('header-bg');
+  } else {
+    header.classList.remove('header-bg');
+  }
+});
+menuBtn.addEventListener('click', () => {
+  menuLinks.style.display = 'flex';
+  menuLinks.classList.add('menu-links');
+});
+closeMenuBtn.addEventListener('click', () => {
+  menuLinks.style.display = 'none';
+  menuLinks.classList.remove('menu-links');
+});
+
+const speakerGrid = document.querySelector('.speaker-grid');
+
+function implementFeature() {
+  myFeatures.forEach((feature) => {
+    speakerGrid.innerHTML += `<article class="feature-article">
+    <div class="feature-image">
+      <div class="grid-img"><img src="./assets/grid.jpg" alt="grid image"></div><img src=${feature.featureImage}
+        alt="Feature Image">
+    </div>
```

Comment:
- [ ] The images are losing their width and height in the mobile view. Kindly work on this
![image](https://user-images.githubusercontent.com/33067256/128328282-f4b754d1-0f2b-4da4-a6f6-d8708b4cb4c1.png)

Reply:
The paragraph text is not bold, that's the cocogoose font default font style.

Reply:
- [ ] This issue still exists and causing alignment issues on desktop resolution. Kindly check the screenshot below :arrow_down: 

![Screenshot 2021-08-05 at 21-29-04 LEEHAR'S FASHION WEEK](https://user-images.githubusercontent.com/1642014/128402970-625e7324-2934-4fb9-930f-4072efa52cbb.png)

Reply:
the images are actually of different original sizes and I try to make them work well for the desktop screen. It only shows the big image on the 4k screen
![screenshot (6)](https://user-images.githubusercontent.com/44624138/128425780-9e8cac78-8a24-44af-b01f-86d6bccc2c8f.png)
this is a screenshot of it  on the laptop large screen